configuration/concepts/adult_hiv_followup_form.csv

```diff
@@ -0,0 +1,71 @@
+uuid,Members,Answers,Fully specified name:fr,Fully specified name:en,Short name:fr,Short name:en,Data class,Data type,_version:1,_order:1002
+259735eb-9844-4ce1-b87d-e78b9e4846fd,,,Poids (kg),Weight (kg),Poids (kg),Weight (kg),Misc,Numeric,,

```

Comment:
We shouldn't rename the vitals concepts, and we should re-use the existing ones. I will work on this and ask Lisa to create a separate story to translate them.